projects/js-packages/components/tools/get-site-admin-url/index.ts

```diff
@@ -0,0 +1,26 @@
+declare global {
```

Comment:
Maybe it's a bit cleaner to keep the `declaration` in the separate file, say `types.d.ts`

Reply:
Thanks for the suggestion, I have updated it.